.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,11 +5,6 @@ Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/d
 
 I, the developer opening this PR, do solemnly pinky swear that:
 
-- [ ] I've documented my API changes in Swagger
-
-In all cases:
-
+- [ ] I've documented my API changes in Swagger. Bump `version` field in [API doc](src/main/resources/swagger/api-docs.yaml).
 - [ ] Get a thumbsworth of review and PO signoff if necessary
-- [ ] Verify all tests go green
-- [ ] Squash and merge; Delete your branch after this
-- [ ] Test this change deployed correctly and works on dev environment after deployment
+- [ ] In most cases, keep your pull request branch up-to-date before merging; You may by-pass this with admin privilege if the change is trivial.
```

Comment:
Should we clarify what `trivial` means?

Reply:
Also, should all Callistans have admin privilege?

Reply:
Hmm... why do we want to allow people to not rebase on small changes? Do you mean they don't have to wait for the tests if its a trivial change?

If that's the case, I think `plain text change` is a better descriptor than trivial. 

Do you mean not waiting for the checks to go green? I think its worth keeping something similar to line 13, i.e. `Verify all checks pass`

Sorry for being pedantic, but I think this checklist is pretty important for new contributors/team members